### DISCUSSION TITLE
Missing Artifacts menu from ContextSidebar.vue

### DISCUSSION THREAD

**Discussion by matter1-git** [PrefectHQ/prefect#14542] [General]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Prefect documentation for this issue.
- [X] I checked that this issue is related to Prefect and not one of its dependencies.

### Bug summary

The artifacts menu has vanished from the sidebar, but the route is available upon navigating to `/artifacts` manually.

To fix this, add an item in **p-context-sidebar**, located in `ContextSidebar.vue`:
`<p-context-nav-item title="Artifacts" :to="routes.artifacts()" />`

### Reproduction

```python3
fire up prefect with prefect dev start
```

### Error

```python3
N/A
```

### Versions (`prefect version` output)

```Text
3.0.0rc10
```

### Additional context

N/A

**Comment by aaazzam** [comment]
👋 @matter1-git! This was a deliberate choice from #13226. 

>The goal here is to reduce the # of forks at the highest level of the app. While artifacts can be created and exist outside the context of flows, they rarely are and so we want the UI to better reflect the relationship that artifacts do have - namely with flow runs and task runs. After these changes, artifacts are still available in the UI through the flow run graph on the flow run page. Their individual pages are also still available and exposed throughout the app but just not via top-level site navigations.

While this choice was intentional, _genuinely_ open to feedback if you're using artifacts in a way that this choice undermines. I want to hear from others too and to that end, I'm going to convert this to a discussion.

**Comment by mattinwood** [comment]
We plan to use artifacts to elevate certain key data about deployments and runs for non-technical users, and would like to have a quick way to link to the artifacts page (currently accessible through site search) as a "one stop shop" for navigating that information. This would allow our users to leverage artifact descriptions to find the correct information rather than sifting through the larger list of deployments, flows, and runs.

**Comment by varennes** [comment]
Not having the artifacts linked in the flow run is bad UI.

Searching for one particular artifact on the Artifact page is a bad user experience.
At my company, many flow runs produce artifacts with the same name and searching for them from the Artifacts page is an extremely bad experience. I cant even search by flow-run name to filter artifacts.

**Reply by varennes** [reply]
Only having artifacts in the flow graph is not a good solution.

For complicated graphs that produce many artifacts, they get lost in the graph visualization.

**Comment by x86demon** [comment]
Would it be possible to return the Artifacts menu to the sidebar?
It may be hidden by default, but it would be helpful to have the option to enable it. We use Artifacts to create data synchronization reports, but they are now difficult to find because we have a complex flow structure with multi-level subflows and tasks.
